packages/main/src/webComponents/ViewSettingsDialog/index.tsx

```diff
@@ -14,10 +14,10 @@ interface ViewSettingsDialogAttributes {
 
 export interface ViewSettingsDialogDomRef extends ViewSettingsDialogAttributes, Ui5DomRef {
   /**
-   * Sets a JavaScript object, as settings to the `ViewSettingsDialog`. This method can be used after the dialog is initially open, as the dialog need to set its initial settings. The `ViewSettingsDialog` throws an event called "before-open", this can be used as trigger point. The object should have the following format: `{ { "sortOrder" : "Ascending", "sortBy" : "Name", "filters" : [{"Filter 1": ["Some filter 1", "Some filter 2"]}, {"Filter 2": ["Some filter 4"]}]} }`
-   * @param {string} settings - A value to be set as predefined settings.
+   * Sets a JavaScript object, as settings to the `ViewSettingsDialog`. This method can be used after the dialog is initially open, as the dialog need to set its initial settings. The `ViewSettingsDialog` throws an event called "before-open", this can be used as trigger point. The object should have the following format: `{sortOrder: "Ascending", sortBy: "Name", filters: [{Filter 1: ["Some filter 1", "Some filter 2"]}, {Filter 2: ["Some filter 4"]}]}`
+   * @param {Object} settings - A value to be set as predefined settings.
```

Comment:
Optional:
In my opinion, JSDoc and the description here can be improved as well. As we only adopt the description from the UI5 Web Components, this has to be adapted in their repo. Otherwise all changes will be lost when we run our script again. (e.g. if `@ui5/web-components` update their version).

That's why I'm just making a suggestion here and if you find it better that way too, you can add this suggestion to the PR in the UI5 web components repo.

```suggestion
   * @param {Object} settings - predefined settings.
   * @param {string} settings.sortOrder - sort order
   * @param {string} settings.sortBy - sort by
   * @param {Array.<Object>} settings.filters - filters
```